server/routes/overview/viewGoalsController.ts

```diff
@@ -1,11 +1,29 @@
 import { RequestHandler } from 'express'
 import ViewGoalsView from './viewGoalsView'
+import EducationAndWorkPlanService from '../../services/educationAndWorkPlanService'
 
 export default class ViewGoalsController {
-  constructor() {}
+  constructor(private readonly educationAndWorkPlanService: EducationAndWorkPlanService) {}
```

Comment:
No need for the constructor and no need to inject the service into the controller